tools/check_format.py

```diff
@@ -24,6 +25,16 @@ def printError(error):
   print "ERROR: %s" % (error)
 
 
+def checkNamespace(file_path):
+  with open(file_path) as f:
+    text = f.read()
+    if not re.search('^\s*namespace\s+Envoy\s*{', text, re.MULTILINE) and \
```

Comment:
Clang-tidy will figure out the right number of spaces. I just wanted to ensure that the line was present at all.